sources/models/src/lib.rs

```diff
@@ -79,6 +79,11 @@ The `#[model]` attribute on Settings and its sub-structs reduces duplication and
 * [Model](src/aws-ecs-2/mod.rs)
 * [Default settings](src/aws-ecs-2/defaults.d/)
 
+## aws-ecs-2: Amazon ECS
+
+* [Model](src/aws-ecs-2/mod.rs)
```

Comment:
This will need to reference the `ecs-1` path. When this is in the rendered html view on GitHub, it is not able to follow the symlink from `ecs-2` to `ecs-1` where the actual file exists.